src/db/db.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
This is what is throwing the errors when you first load the page, as there is nothing here to load. Consider putting in  a pre-filled note or something to remove those errors.